### PR TITLE
Fix testsuite when GMP is not present

### PR DESCRIPTION
**Pull request by sloriot**
The fix is not perfect has it only checks for GMP presence but it will fix the testsuite.

### CODE REVIEW

File: Arrangement_on_surface_2/test/Arrangement_on_surface_2/cgal_test.cmake [hunk 1]
```diff
@@ -873,6 +873,10 @@ endfunction()
 #---------------------------------------------------------------------#
 function(test_polycurve_conic_traits)
 #  echo polycurve test starting
+  if($ENV{CGAL_DISABLE_GMP})
+    MESSAGE(STATUS "test_polycurve_conic_traits requires CORE and will be executed")
```
Comment (lrineau) [inline Arrangement_on_surface_2/test/Arrangement_on_surface_2/cgal_test.cmake]: > will **not** be executed

**Comment by sloriot** [comment]
Amended. Thanks.

**Comment by sloriot** [comment]
The compilation errors are gone. We are now left with runtime errors...

**Comment by lrineau** [comment]
Anyway, the results of CGAL-4.11-Ic-59 were too incomplete, and I have repushed the `integration` branch of yesterday (plus new merges).

**Comment by lrineau** [comment]
@afabri A lot of Windows results of Periodic_3 did not finish before the timeout, but those that has finished did finish in a lot less than the time bound. That is strange. Could you have a look?

https://cgal.geometryfactory.com/CGAL/testsuite/results-4.11-Ic-61.shtml#Periodic_3_triangulation_3

**Comment by lrineau** [comment]
Sorry, wrong PR.

**Pull request merged**